docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -1,275 +1,459 @@
+                                 ||| % $._config, legendFormat='{{%(clusterLabel)s}} Receive' % $._config
+                               ))
+                               .addTarget(prometheus.target(
+                                 |||
+                                   sum(
+                                       instance:node_network_transmit_bytes_excluding_lo:rate%(rateInterval)s{%(nodeExporterSelector)s}
+                                   ) by (%(clusterLabel)s)
+                                 ||| % $._config, legendFormat='{{%(clusterLabel)s}} Transmit' % $._config
+                               ))
+                             )
+                             .addPanel(
+                               networkSaturation
+                               .addTarget(prometheus.target(
+                                 |||
+                                   sum(
+                                       instance:node_network_receive_drop_excluding_lo:rate%(rateInterval)s{%(nodeExporterSelector)s}
+                                   ) by (%(clusterLabel)s)
+                                 ||| % $._config, legendFormat='{{%(clusterLabel)s}} Receive' % $._config
+                               ))
+                               .addTarget(prometheus.target(
+                                 |||
+                                   sum(
+                                       instance:node_network_transmit_drop_excluding_lo:rate%(rateInterval)s{%(nodeExporterSelector)s}
+                                   ) by (%(clusterLabel)s)
+                                 ||| % $._config, legendFormat='{{%(clusterLabel)s}} Transmit' % $._config
+                               ))
+                             )
+                           )
+                           .addRow(
+                             row.new('Disk IO')
+                             .addPanel(
+                               diskIOUtilisation
+                               .addTarget(prometheus.target(
+                                 |||
+                                   sum(
+                                       instance_device:node_disk_io_time_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}
+                                       / scalar(count(instance_device:node_disk_io_time_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}))
+                                   ) by (%(clusterLabel)s, device)
+                                 ||| % $._config, legendFormat='{{%(clusterLabel)s}} {{device}}' % $._config
+                               ))
+                             )
+                             .addPanel(
+                               diskIOSaturation
+                               .addTarget(prometheus.target(
+                                 |||
+                                   sum(
+                                     instance_device:node_disk_io_time_weighted_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s}
```

Comment:
```suggestion
                                     (instance_device:node_disk_io_time_weighted_seconds:rate%(rateInterval)s{%(nodeExporterSelector)s} != 0)
```
Since for USE method we are not interested in zero values, what about doing the above? It should declutter legend even more.